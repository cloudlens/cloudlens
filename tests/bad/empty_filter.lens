stream when (foo bar) {
    print(e.hi)
}
