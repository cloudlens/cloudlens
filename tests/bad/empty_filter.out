Empty <when> filter. Maybe you forgot the entry name.
